.github/workflows/e2e_cloud.yml

```diff
@@ -38,6 +38,7 @@ jobs:
         run: |
           curl -fsS https://packages.fluvio.io/v1/install.sh | bash
           echo 'export PATH="${HOME}/.fluvio/bin:${PATH}"' >> ~/.bashrc
+          source ~/.bashrc
```

Comment:
- run: echo "$HOME/.fluvio/bin" >> $GITHUB_PATH

Reply:
here: [818b30d](https://github.com/infinyon/http-connector/pull/31/commits/818b30d281977235e1ff5011a54f5e6953ca3c3b)

Reply:
Mmm: https://github.com/infinyon/http-connector/actions/runs/4704457728/jobs/8344100026#step:3:38

Reply:
Moving out of Makefile..

Reply:
Didn't thought about the `$` syntax concerns..

Reply:
Installation went through successfully.